solax/utils.py

```diff
@@ -75,3 +78,16 @@ def twoway_div10(val, *_args, **_kwargs):
 
 def twoway_div100(val, *_args, **_kwargs):
     return to_signed(val, None) / 100
+
+
+def sum_of_list_not_zero(value: List[Number]):
+    """Validate that at least one element
+
+    Args:
+        value (List[Number]): list to validate
+
+    Raises:
+        Invalid: if all elements are zero
+    """
+    if len(value) != 0 and sum(value) == 0:
+        raise Invalid("All elements in the list are zero")
```

Comment:
This will assert the list has all zeros even if the list is `[1, -1]`

Reply:
probably want something with `any()`

Reply:
Yes, I assumed that the data array does not contain negative values. Changed it..